content/_data/changelogs/weekly.yml

```
@@ -22006,6 +22006,9 @@
 
   - version: '2.432'
     date: 2023-11-14
+    banner: >
+      The Windows container images of this release switch from a windowsservercore-1809 Temurin base image to a windowsservercore-ltsc2019 Microsoft base image.
+      Note also that a proper set of tags is now published, and they include "ltsc2019" instead of only "2019".
```

```suggestion
      Note also that a proper set of tags is now published and they include "ltsc2019" instead of only "2019".
```
small nit suggestion, the `,` isn't necessary here